src/components/organisms/Supporters.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {
+  Box,
+  Grid
+} from "@material-ui/core";
+
+import Logos from '../atoms/Logos' ;
+import SectionHeading from '../atoms/SectionHeading';
+
+
+function Supporters(props) {
+  const { supporters } = props
+  return (
+    <Box component="section">
+      <Grid container justify="center" alignItems="center" spacing={5}>
+        
+        <Grid item sm={12}>
+          <SectionHeading title="With support from"/>
+        </Grid>
+        <Grid container justify="center" alignItems="center" spacing={5}>
+        <Grid
+          item
+          container
+          spacing={5}
+          justify="center"
+          className="why-cards-section"
+        >
+          {supporters.map((logos, key) => {
+            return (
+              <Grid item>
+                <Logos key={key} logos={logos} />
```

Comment:
I might be missing something here, but what's this component for? Is this for showing the logo for each supporter? Can one support have multiple logos? Or is it just one logo?

Reply:
Each supporter has just one logo. I used this component to handle organizing the logos.

Reply:
Hmmm, I think we can probably simplify this. I think in terms of layout we have a title ("With Support From") and then a grid containing the logos of the sponsors. We are using both `Box` and `Grid` `mui` components as containers. 

The `mui` `Grid` component is nice because it handles reorienting multiple components in a grid layout (particularly across different sized devices). So I think we only want to use it when we have multiple items in our container. We also can get all the functionality we need out of it with just one `Grid container` and a series of `Grid items`. In other cases we can just use `Box` as a generic mui container. 

If we follow this logic, we can actually just hoist the `img` from `Logos` and put it into this component, removing the other grid items in that component. So I think we would end up with this:

```suggestion
function Supporters(props) {
  const { supporters } = props
  return (
    <Box component="section">
        <Box justifyContent="center" alignItems="center" >
          <SectionHeading title="With support from" />
        </Box>
        <Grid container justifyContent="center" alignItems="center" spacing={5} >
          {supporters.map((logo) => (
            <Grid item>
              <img
                src={logo.featureImage}
                alt="right-on-education-product-features"
                style={{  height: '130px', width: '350px', marginLeft: "60px", marginRight: "60px" }}
              />
            </Grid>
          ))}
        </Grid>
     
    </Box>
  );
}

```
(note that the above suggestion replaces everything from lines 11 - 40 in the original Supporters.jsx and allows Logos.jsx to be deleted. 

@cindyqquach maybe give this a shot and see what you think

Reply:
I update the code and it works fine, I just had to adjust the height and width a little. Should be good now!

Reply:
Sorry Jason, got confused on who was working on what here. Thanks for addressing this.